lib/rules/hidden-content.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": "hidden-content",
+  "matches": "hidden-content.js",
+  "tags": [
+    "experimental",
+    "review-item"
+  ],
+  "metadata": {
+    "description": "Informs users about hidden content.",
+    "help": "Manually check elements that are hidden with display: none, visibility: hidden, or aria-hidden: \"true\""
```

Comment:
There are two ways to deal with this

1. trigger the UI state that displays the content and then analyze, or
1. manually assess it (not sure how you would do that without first doing the above)